azext_iot/common/auth.py

```diff
@@ -25,3 +28,21 @@ def get_aad_token(cmd, resource=None):
         "subscription": subscription,
         "tenant": tenant,
     }
+
+
+def generate_device_key(primarykey, device_id):
```

Comment:
@digimaun - is there a better place for this?

Reply:
I think we should carve out a utility area in the `azext_iot/dps` directory.

Reply:
moved it to common / utility -since it is shared across central and dps